docs/providers.md

```diff
@@ -86,6 +88,24 @@ This command undeploys the app in Docker. For the given namespace, the undeploy:
 
 If `--name` was provided in the Docker artifacts file, this command will not work.
 
+### Docker Compose
+
+**Features**
+
+The Docker Compose provider: ``docker-compose``
+will start a set of docker containers for a multi-tier application as
+specified in a Docker Compose YAML file on the host the Atomic App is
+deployed. Since, a docker compose YAML file will represent
+the entire graph of applications in a Nulecule file, you don't need to run
+docker compose on each application in the Nulecule graph. However, since
+each provider for an application must point to an artifact file, you can
+point to an empty ``artifacts/docker-compose/noop-compose.yaml`` artifact
+file for applications you want to skip running docker compose on.
+
```

Comment:
Because, artifacts/source are a must for a graph item. And, an entry in ``artifacts`` must point to a valid file. In case of compose, a single file contains instructions to deploy a multi container application.